easybuild/easyconfigs/e/elbencho/elbencho-2.0-3-GCC-10.3.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = "MakeCp"
+
+name = "elbencho"
+version = "2.0-3"
+
+homepage = "https://github.com/breuner/elbencho"
+description = """ A distributed storage benchmark for files, objects & blocks with support for GPUs"""
+
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+dependencies = [('Boost', '1.76.0'), ('libaio', '0.3.112')]
+
+sources = [{
+    'filename': "v%(version)s.tar.gz",
+    'git_config': {
+        'url': 'https://github.com/breuner',
+        'repo_name': 'elbencho',
+        'tag': 'v%(version)s'
+    },
+
+}]
+checksums = ['f6c34f46b61451bcb1362e73d7754f307ae972c467b135e3279fd477586b5b7d']
+
```

By default, the `make` of elbancho ignores the build environment defined by EasyBuild. It needs to be explicitly specified with the following `buildopts`:
```suggestion
buildopts = 'BUILD_VERBOSE=1 CXXFLAGS_EXTRA="$CXXFLAGS" LDFLAGS_EXTRA="$LDFLAGS"'

```